src/Files.App/Views/WidgetsPage.xaml.cs

```diff
@@ -61,15 +59,21 @@ private void ViewModel_WidgetListRefreshRequestedInvoked(object? sender, EventAr
 
 		public void ReloadWidgets()
 		{
-			quickAccessWidget = WidgetsHelpers.TryGetWidget<QuickAccessWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadQuickAccessWidget, quickAccessWidget);
-			drivesWidget = WidgetsHelpers.TryGetWidget<DrivesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadDrivesWidget, drivesWidget);
-			bundlesWidget = WidgetsHelpers.TryGetWidget<BundlesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadBundles, bundlesWidget);
-			fileTagsWidget = WidgetsHelpers.TryGetWidget<FileTagsWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadFileTags, fileTagsWidget);
-			recentFilesWidget = WidgetsHelpers.TryGetWidget<RecentFilesWidget>(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadRecentFiles, recentFilesWidget);
+			quickAccessWidget = WidgetsHelpers.TryGetWidget(UserSettingsService.PreferencesSettingsService, Widgets.ViewModel, out bool shouldReloadQuickAccessWidget, quickAccessWidget);
```

Comment:
@d2dyno1 are you okay with this?

Reply:
FYI, certainly sure.

```cs
public static TWidget? TryGetWidget<TWidget>(
    IPreferencesSettingsService preferencesSettingsService,
    WidgetsListControlViewModel widgetsViewModel,
    out bool shouldReload, TWidget? defaultValue = default)
        where TWidget : IWidgetItemModel, new()
```

Adding target type like TryGetWidget<TWidget>() is redundant because type will be discovered by the last parameter `TWidget? defaultValue = default`